src/pages/account/organization/new/banner/Banner.component.js

```diff
@@ -20,7 +20,7 @@ const Banner = () => {
       </ul>
       <p>
         更多详细信息，点击
-        <Link href="https://asktug.com/t/topic/93140">认证前 vs 认证后权益对比</Link>，
+        <Link href="https://tidb.io/account/organization/benefits">团队认证权益</Link>，
```

Comment:
Please use relative path `/account/organization/benefits` because NextJS (`Link` Component) will do partially rendering and prefetch optimization if it's a relative path.